sdk/core/azure-core/src/main/java/com/azure/core/http/policy/FixedDelayOptions.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.policy;
+
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * The configuration for exponential backoff.
+ */
+public class FixedDelayOptions {
+    private Integer maxRetries;
+    private Duration delay;
+
+    /**
+     * Gets the max retry attempts that can be made.
+     *
+     * @return The max retry attempts that can be made.
+     */
+    public Integer getMaxRetries() {
+        return maxRetries;
+    }
+
+    /**
+     * Sets the max retry attempts that can be made.
+     *
+     * @param maxRetries the max retry attempts that can be made.
+     *
+     * @return The updated {@link FixedDelayOptions}
+     */
+    public FixedDelayOptions setMaxRetries(Integer maxRetries) {
```

Comment:
minor: the behavior for FixedDelay strategy is: no defaults and you can only set tries and delay together. 
With this in mind, could FixedDelayOptions have some sort of similar behavior so users would know sooner when they apply invalid configuration?